app/mailers/billing_mailer.rb

```diff
@@ -0,0 +1,14 @@
+class BillingMailer < ApplicationMailer
+
+  # Subject can be set in your I18n file at config/locales/en.yml
+  # with the following lookup:
+  #
+  #   en.billing_mailer.billing.subject
+  #
+  def billing hiring, user
+    @hiring = hiring
+    @user = user
+
+    mail to: @user.email
```

Comment:
c nghĩ e nên viết ntn. mail k có subject ah?
@user có dùng ở view đâu mà sử dụng biến instance
mail(to: user.email, subject: "subject cua em") do |format|
 format.html{render locals: billing_locals(hiring)}
end

private
def billing_locals hiring
{
  hiring_time: hiring.time,
...
}

ở template mail chỉ việc gọi hiring_time để hiển thị thôi
end

Reply:
Cách này hay chị ạ :). để em sửa luôn

Reply:
lần sau làm thế nhé. đừng dùng biến @